File: TestSets.java
import java.util.HashSet;

public class TestSets
{
    private enum Color {
        RED, GREEN, BLUE, BLACK
    }

    public static class ColorSet extends HashSet<Color> {}
    private static class digits extends HashSet<Integer> {}
    private static class letters extends HashSet<Character> {}

    private static HashSet<Integer> smallNums;
    private static digits nameddigits;
    private static HashSet<Integer> unnameddigits;
    private static HashSet<Boolean> flags;

    public static void main(String[] args)
    {

        System.out.println("Done!");
    }
}

File: TestSets.pas
PROGRAM TestSets;

TYPE
    digits = SET OF 0..9;
    letters = SET OF char;
    Color = (Red, Green, Blue, Black);
    ColorSet = SET OF Color;

    digits_wrong  = SET TUPLE;
    letters_wrong = SET char;
    reals_wrong   = SET OF real;
VAR
    smallNums: SET OF 1..100;
    namedDigits : digits;

    unnamedDigits : SET OF 1..5;
    flags : SET OF boolean;


BEGIN
END.
